src/main/java/org/opentripplanner/raptor/rangeraptor/path/ReversePathMapper.java

```diff
@@ -50,6 +50,8 @@ public ReversePathMapper(
 
   @Override
   public RaptorPath<T> mapToPath(final DestinationArrival<T> destinationArrival) {
+    // TODO: 2023-05-11 via pass through: do we need to include c2 here?
```

Comment:
Probably yes?

Reply:
We actually don't need to include c2 in ReversePathMapper since multi-factor raptor does not support reverse search.